src/components/Card.js

```diff
@@ -1,14 +1,29 @@
-import React from 'react'
+import React from "react";
 
-const Card = (props) => {
-  console.log(props);
-  return (
-    <div>
-   <div ClassName = "card">
-    <img src= {`../images/${props.img}`} alt ="" className = "card--image"/>
-    </div>
-      </div>
-  )
+export default function Card({ cardImages }) {
+	return (
+		<>
+			<div className="grid grid-cols-4 gap-4">
+				<div className="col-span-4 ...">
+					<div className="grid grid-cols-2 gap-4">
+						<div ClassName="...">
```

Comment:
please use appropriate classnames that are in `kebab-case`

Reply:
so, what I meant here is when you're defining class names, you need to epxress them in the `kebab-case` format. The only time you'd use a different case is perhaps if you were using css or sass modules. Then you would use `camelCase`
if you see there, you defined a class as `class="..."`